checkstyle.xml

```diff
@@ -185,12 +185,12 @@
              value="GenericWhitespace ''{0}'' is not preceded with whitespace."/>
         </module>
         <module name="Indentation">
-            <property name="basicOffset" value="2"/>
+            <property name="basicOffset" value="4"/>
```

Comment:
I would recommend removing all these indentation properties, because these settings values already are the default value settings in the checkstyle. See [Default Indentation Properties](https://checkstyle.sourceforge.io/config_misc.html#Indentation_Properties).

Reply:
Done